### ISSUE TITLE
java.lang.NumberFormatException

### ISSUE THREAD

**Issue by hatpick** [nulab/zxcvbn4j#49]
I get the following, I don't know what the password is. Can you add an exception handling to your code? or maybe use regex to detect the problematic chars and don't run this matcher.

```
java.lang.NumberFormatException: For input string: "5
"
at com.nulabinc.zxcvbn.matchers.DateMatcher.execute Line 41
```

Thank you.

**Comment by SteveLeach-Keytree** [comment]
@hatpick - could you check whether version 1.3.0 fixes your problem? I'm hoping it does.

**Comment by vvatanabe** [comment]
This issue has been fixed in version 1.3.0.
